src/poetry/utils/env.py

```diff
@@ -481,7 +487,8 @@ def activate(self, python: str, io: "IO") -> "Env":
                         [
                             python,
                             "-c",
-                            "\"import sys; print('.'.join([str(s) for s in sys.version_info[:3]]))\"",
+                            "\"import sys; print('.'.join([str(s) for s in"
+                            ' sys.version_info[:3]]))"',
```

Comment:
I wonder if we should keep the original format here for better readability. What do you think? (same for the lines 730 and 805, 859)

This can be achieved by `#fmt: off/on` (https://stackoverflow.com/a/58584557/9750706)

Reply:
In this case, because the exact same string is repeated 4 times in this class, I'd suggest defining a string constant in the class (next to `ENVS_FILE`) called something like `VERSION_SCRIPT`.

Then we won't have to wrap it, and won't have to fiddle with black formatting.

Reply:
Good idea  :+1: I saw there is already a `GET_PYTHON_VERSION` at the top of the file. I think we can use this one.

Reply:
Even better! Adjusted the last commit to use the existing constant.